test/include/seqan3/test/performance/sequence_generator.hpp

```diff
@@ -78,15 +78,21 @@ auto generate_sequence_seqan2(size_t const len = 500,
                               size_t const variance = 0,
                               size_t const seed = 0)
 {
+    size_t max_val;
+    if constexpr (std::unsigned_integral<alphabet_t>)
+        max_val = std::numeric_limits<size_t>::max();
```

Comment:
I can't follow this change. When alphabet_t is an integral then we always use size_t? But that would exceed the max value if alphabet_t is uint32_t? What am I a missing. Also in SeqAn `seqan::ValueSize<uint64_t>::VALUE` returns 0, which should be -1  which should be std::numeric_limits<uint64_t>::max(), or not? Could you elaborate?